playbooks/roles/minishift/tasks/main.yml

```
@@ -74,9 +79,19 @@
   when: ((minishift_profile_status.stdout_lines and minishift_profile_status.stdout_lines[0]|lower == "does not exist") or
         (minishift_profile_status.rc != 0)) and (minishift_external_vm_ip == "")
 
-# Start minishift on an external VM if configured
+# Start minishift on an external VM if configured, then shut it down (if it was configured not to start, but we wanted the profile to exist)
 - name: "Initialization of minishift cluster with profile {{ profile }} on an existing VM ip {{ minishift_external_vm_ip }}"
-  shell: "{{ minishift_bin }} start --profile {{ profile }} --openshift-version {{ oc_version }} --vm-driver generic --remote-ipaddress {{ minishift_external_vm_ip }} --remote-ssh-user {{ minishift_external_vm_user }} --remote-ssh-key {{ minishift_external_vm_ssh_key_location }}"
   when: ((minishift_profile_status.stdout_lines and minishift_profile_status.stdout_lines[0]|lower == "does not exist") or
         (minishift_profile_status.rc != 0)) and (minishift_external_vm_ip != "")
-
+  block:
+    - name: "Initialize minishift"
+      shell: "{{ minishift_bin }} start --profile {{ profile }} --openshift-version {{ oc_version }} --vm-driver generic --remote-ipaddress {{ minishift_external_vm_ip }} --remote-ssh-user {{ minishift_external_vm_user }} --remote-ssh-key {{ minishift_external_vm_ssh_key_location }}"
+
+    - name: "Stop the minishift profile {{ profile }}"
+      shell: "{{ minishift_bin }} stop --profile {{ profile }}"
+      when: (start_minishift|bool == false)
+
+- name: "Running cleanup post-tasks {{ minishift_post_hooks }}"
```

I'm guessing this should be 
```suggestion
- name: "Running minishift post-tasks {{ minishift_post_hooks }}"
```